images/udevd/scripts/startup.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -e
 
-function start_udev() {
+function start_udev {
```

Comment:
isn't the `function` keyword itself a bash-ism?  i thought the POSIX style was `start_udev() {`...

Reply:
Sorry, yea. I get them confused. Just looked it up and yea, I had this around the wrong way. `start_udev() {` is POSIX and `function start_udev {` is a compatibility with ksh. I'll switch these around.